backend/battles/models.py

```diff
@@ -1,12 +1,14 @@
 from django.db import models
 
-from pokemon.models import Pokemon  # noqa
-from users.models import User  # noqa
+from pokemon.models import Pokemon
+from users.models import User
 
 
 class Battle(models.Model):
+    timestamp = models.DateTimeField(auto_now_add=True, null=True)
     creator = models.ForeignKey(User, on_delete=models.CASCADE, related_name="battles_as_creator")
     opponent = models.ForeignKey(User, on_delete=models.CASCADE, related_name="battles_as_opponent")
+    status = models.CharField(max_length=20, null=True)
```

Comment:
As you are dealing with `CharField` here, I would suggest the change bellow, because the `null=True` isn't doing anything in this case. [Here](https://stackoverflow.com/a/8609425) you can read a better explanation about the use of `null` and `blank` and how it works with text fields.

```suggestion
    status = models.CharField(max_length=20, blank=True)
```